drivers/hmis/lib/hmis_util/service_types.rb

```diff
@@ -34,5 +31,111 @@ def self.seed_hud_service_types(data_source_id)
         end
       end
     end
+
+    HUD_SERVICE_INSTANCE_CONFIG = [
+      {
+        record_type: 200, # BedNight
+        project_types: [
+          1, # ES
+          0, # ES NBN (2024)
+        ],
+      },
+      {
+        record_type: 141, # P1 Services Provided - PATH Funded
+        project_types: [
+          4, # Street Outreach
+          6, # Services Only
+        ],
+        funders: [21], # HHS: PATH - Street Outreach & Supportive Services Only
+      },
+      {
+        record_type: 161, # P2 Referrals Provided - PATH
+        project_types: [
+          4, # Street Outreach
+          6, # Services Only
+        ],
+        funders: [21], # HHS: PATH - Street Outreach & Supportive Services Only
+      },
+      {
+        record_type: 142, # R14 RHY Service Connections
+        project_types: [
+          0, # ES
+          1, # ES
+          2, # TH
+          6, # services only
+          12, # HP
+        ],
+      },
+      {
+        record_type: 143, # W1 Services Provided – HOPWA
+        project_types: [
+          0, # Emergency Shelter
+          1, # Emergency Shelter
+          2, # Transitional Housing
+          3, # PH - Permanent Supportive Housing
+          6, # Services Only
+          12, # Homelessness Prevention
+        ],
+      },
```

Comment:
So this feels kinda wonky, but this is the config for which service types should be available for which project types & funders. I didn't get to implementing the funder piece yet, I'm going to pull that to another PR since we don't currently have support for that in Form Instances.

The seed will create `Instance`s that indicate: "use the `service` form for service type X in project type Y"

Reply:
I don't have a problem with this setup, except maybe each one should be its own constant and then `HUD_SERVICE_INSTANCE_CONFIG` could just be an array of all the possible situations?  Might make maintenance and re-use easier in the future.  
One question though, are we enforcing that if you choose record type 143, you must apply it to one of the project types in the list?  (I think that's probably good, just checking.)

Reply:
Okay. I think I understand your question now, and the answer is no. If you wanted to go in manually and say "Enable HOPWA services for all Rapid Re-Housing Projects" there's nothing stopping you from creating an Instance like that (in the rails console / future instance-manager).